<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportItem</class>
 <widget class="QDialog" name="ExportItem">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export an Item</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::InputMethodHint::ImhExclusiveInputMask</set>
  </property>
  <widget class="QPushButton" name="btnExport">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtID">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="lblInfo">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtExportCount">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Export Quantity</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>125</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Export Date</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateExport">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>110</width>
     <height>29</height>
    </rect>
   </property>
   <property name="maximumDateTime">
    <datetime>
     <hour>23</hour>
     <minute>59</minute>
     <second>59</second>
     <year>9999</year>
     <month>12</month>
     <day>30</day>
    </datetime>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::Section::DaySection</enum>
   </property>
   <property name="displayFormat">
    <string>d/M/yyyy</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>621</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
